webapp/components/test-runs.html

```diff
@@ -75,30 +75,31 @@
                 }))
             );
           }
-          const fetches = await Promise.all(runs);
-
-          // Filter unresolved fetches and flatten any array-fetches into the array.
-          const nonEmpty = fetches.filter(e => e);
-          const flattened = nonEmpty.reduce((sum, item) => {
-            return sum.concat(Array.isArray(item) ? item : [item]);
-          }, []);
-          this.testRuns = flattened;
-          return flattened;
+          return Promise.all(runs).then(fetches => {
```

Comment:
Same here. This change seems to be just converting an async function to a non-async one. If so, let's revert the change in this function as well to keep this PR simple (and for my preference of `async`).